{"id":1546,"details":"\n  \n  \u003ctable\u003e\n    \u003ctr\u003e\n      \u003ctd\u003eRegion\u003c/td\u003e\n\u003ctd\u003eGrand Est\u003c/td\u003e\n    \u003c/tr\u003e\n    \u003ctr\u003e\n\u003ctd\u003eDepartment\u003c/td\u003e\n\u003ctd\u003eArdennes\u003c/td\u003e\n\u003c/tr\u003e\n    \u003ctr\u003e\n\u003ctd\u003eCity\u003c/td\u003e\n\u003ctd\u003eBogny-Sur-Meuse\u003c/td\u003e\n\u003c/tr\u003e\n    \u003ctr\u003e\n\u003ctd\u003eReason of Infection\u003c/td\u003e\n\u003ctd\u003eVisit family in Oise\u003c/td\u003e\n\u003c/tr\u003e\n  \u003c/table\u003e\n  ","patient":"88 Years Old Male Patient From Grand Est, Ardennes, Bogny Sur Meuse","source":"unknown_source","status":"in_hospital","confirmed_at":"2020-03-08T00:00:00.000Z","created_at":"2020-03-16T10:13:17.879Z","updated_at":"2020-03-16T10:42:45.348Z","infected_by":"unknown_infection","number":1272,"urls":["https://github.com/lperez31/coronavirus-france-dataset"],"recovered_at":null,"gender":"male","age":88,"nationality":"unidentified_nationality","symptomatic_at":null,"slug":"france-case-1272-88-years-old-male-patient-from-grand-est-ardennes-bogny-sur-meuse","suggested_url":"france-case-1272-88-years-old-male-patient-from-grand-est-ardennes-bogny-sur-meuse","hospital_stay_length_in_days":null,"days_from_symptoms_to_confirmation":null,"without_symptoms":false,"reporting_country":"fr"}